### PR TITLE
Add system scopes to test IDP

### PR DESCRIPTION
**Pull request by feordin**
## Description
A small change to add system scopes to the test IDP.

## Related issues
Addresses [issue AB#104427].

## Testing
By adding these scopes, manual testing is possible.

## FHIR Team Checklist
- **Update the title** of the PR to be succinct and less than 65 characters
- **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- Tag the PR with the type of update: **Bug**, **Build**, **Dependencies**, **Enhancement**, **New-Feature** or **Documentation**
- Tag the PR with **Open source**, **Azure API for FHIR** (CosmosDB or common code) or **Azure Healthcare APIs** (SQL or common code) to specify where this change is intended to be released.
- [ ] CI is green before merge [Build Status](https://microsofthealthoss.visualstudio.com/FhirServer/_build/latest?definitionId=27&branchName=main) 
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

File: src/Microsoft.Health.Fhir.Shared.Web/DevelopmentIdentityProviderRegistrationExtensions.cs [hunk 1]
```diff
@@ -171,8 +171,10 @@ private static IReadOnlyCollection<ApiScope> GenerateSmartClinicalScopes()
 
             scopes.Add(new ApiScope("patient/*.*"));
             scopes.Add(new ApiScope("user/*.*"));
+            scopes.Add(new ApiScope("system/*.*"));
```
Comment (mikaelweave) [inline src/Microsoft.Health.Fhir.Shared.Web/DevelopmentIdentityProviderRegistrationExtensions.cs:174]: Can we add `system/*.read` here? That's the default scope for Inferno section 7.

CleanShot 2023-08-18 at 10 12 38@2x

Comment (feordin) [reply src/Microsoft.Health.Fhir.Shared.Web/DevelopmentIdentityProviderRegistrationExtensions.cs:174]: Added!

**Review by mikaelweave — APPROVED** [review]
Looks good!

**Pull request merged**